geosys/ui/widgets/geosys_coverage_downloader.py

```diff
@@ -187,6 +197,10 @@ def run(self):
                     results = searcher_client.get_catalog_imagery(
                         geometry, self.crop_type, self.sowing_date,
                         filters=self.filters)
+                elif self.map_product == YGM['key'] or self.map_product == YVM['key'] or self.map_product == SAMZ['key']:
```

Comment:
We can simplify this statement and other similar statements from into
```suggestion
                elif self.map_product in [ YGM['key'], YVM['key'], SAMZ['key'] ]:
```

Reply:
@Samweli Did a double-check, those thumbnails definetely makes use of the correct URL for YPM and YGM. The likely reason it looks so similar is because for both cases Helene asked me to make use of an average yield of 80 for the thumbnails. Take into account that both YGM and YPM are yield-based datasets, and both has a 'historical-yield-average' which equals 80 for the thumbnails. When the map creation step is done, a person can provide an average yield value. This is how Helene asked me to put it together.